src/cascadia/UnitTests_Control/ControlInteractivityTests.cpp

```diff
@@ -111,18 +111,20 @@ namespace ControlUnitTests
         settings->Opacity(0.5f);
 
         auto [core, interactivity] = _createCoreAndInteractivity(*settings, *conn);
+        auto controlSettings = core->_settings;

```

Comment:
I don't think this is used

Reply:
GAH nope, it isn't. Gotta fix the tests _again_ for the Windows 10 + acrylic test ;_____;